### ISSUE TITLE
Your license has expired

### ISSUE THREAD

**Issue by pbadenski** [atoti/atoti#369]
## Steps to reproduce

In my jupyter (within docker environment) I ran:
```
!conda config --add channels conda-forge
!conda config --add channels https://conda.atoti.io
!conda install --yes atoti atoti-jupyterlab

import atoti as tt

session = tt.create_session()
```

## Actual vs Expected Result

I'm getting `Exception: Your license has expired, contact ActiveViam to get a new license.` where I was expecting this to work fine.

## Environment
- Python: 3.8.6
- Operating System: linux

**Comment by bloussou** [comment]
Hello,
Which version of atoti do you have ? (`from atoti import __version__`.
Also do you have `ACTIVEPIVOT_LICENCE` or `ATOTI_LICENCE` env variable set on your computer ?

**Comment by pbadenski** [comment]
it throws the license exception when trying to check the version:
```
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
<ipython-input-7-2bfbd9da0f4c> in <module>
----> 1 from atoti import __version__

/opt/conda/lib/python3.8/site-packages/atoti/__init__.py in <module>
     44 (_LICENSE_END_DATE, _IS_COMMUNITY_LICENSE) = retrieve_info_from_jar()
     45 
---> 46 check_license(_LICENSE_END_DATE, _IS_COMMUNITY_LICENSE)
     47 
     48 register_active_plugins()

/opt/conda/lib/python3.8/site-packages/atoti/_licensing.py in check_license(end_date, is_community_license)
    152     for output in get_license_outputs(end_date, is_community_license):
    153         if output.output_type == OutputType.EXCEPTION:
--> 154             raise Exception(output.content)
    155         if output.output_type == OutputType.REGULAR:
    156             print(output.content)

Exception: Your license has expired, contact ActiveViam to get a new license.
```

I don't have neither of these variables set:
```
jovyan@e64ea8ddf388:~$ env | grep LICENSE
jovyan@e64ea8ddf388:~$ 
```

**Comment by bloussou** [comment]
does it work with `conda list` ? Can you also run a `pip list`

**Comment by pbadenski** [comment]
Sure.

```
jovyan@e64ea8ddf388:~$ conda list | grep atotia
atoti                     0.5.5                   dc3a2a6    https://conda.atoti.io
atoti-jupyterlab          0.5.5                   dc3a2a6    https://conda.atoti.io
jdk4py                    11.0.9.0                 py38_2    https://conda.atoti.io
```

```
jovyan@e64ea8ddf388:~$ pip list | grep atoti
atoti                         0.5.5
atoti-jupyterlab              0.5.5
```

**Comment by pbadenski** [comment]
That's so weird, I'm pretty sure I just installed this a few days ago. Thanks, I'll check on my side.

**Comment by pbadenski** [comment]
Oh ok, mystery solved, looks like conda solver pulled an older version (namely `0.5.5`). I think it would be good to mention this in installation docs, as isn't very obvious.

**Comment by bloussou** [comment]
To mention the license duration ?

**Comment by pbadenski** [comment]
Yes - especially considering that conda solver might pull an older version unknowingly to the user (me in this example :smile:).

**Comment by bloussou** [comment]
Hello this was resolved in `0.6.2` by https://github.com/activeviam/atoti/pull/2549
